packages/evolution-legacy/src/components/survey/Survey.js

```diff
@@ -173,7 +173,7 @@ export class Survey extends React.Component {
                   containsHtml       = {false}
                 />
               </div>)}
-              <form onSubmit={(e) => { if (e.key === 'enter' || e.which === 13 ) { e.preventDefault(); }}} className="apptr__form" id="survey_form" style={customStyle} autocomplete="off">
+              <form onSubmit={(e) => e.preventDefault() } className="apptr__form" id="survey_form" style={customStyle} autocomplete="off">
```

Comment:
But the prior condition should have been enough to prevent the enter key from submitting the form, no? Why did it submit anyway?

Reply:
it seems the synthetic event triggered by enter key in an input field does not propagate as a key event... The key presed is lost.